<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroJump</class>
 <widget class="QWidget" name="HeroJump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeroJump</string>
  </property>
  <widget class="QLabel" name="label_Hero">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>465</y>
     <width>36</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemy_Slime">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>489</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_04_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_06_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>220</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemy_Oculothorax">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>475</y>
     <width>55</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_02_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_00_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_00_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_options">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>270</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>320</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Score">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QComboBox" name="pushButton_comboBox_ports">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>221</y>
     <width>168</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_04_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_06_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_03_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_05_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_01_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_01_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_03_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_05_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_02_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_09_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_08_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_07_I">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_09_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_08_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_07_O">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Environment_layer_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-230</y>
     <width>928</width>
     <height>793</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>270</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="pushButton_label_menu">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>180</y>
     <width>250</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_score">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>270</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemy_Ogre_Boss">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>227</y>
     <width>285</width>
     <height>285</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemy_Ogre_Shockwave">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>471</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_Environment_layer_10</zorder>
  <zorder>label_Environment_layer_09_O</zorder>
  <zorder>label_Environment_layer_09_I</zorder>
  <zorder>label_Environment_layer_08_I</zorder>
  <zorder>label_Environment_layer_08_O</zorder>
  <zorder>label_Environment_layer_07_I</zorder>
  <zorder>label_Environment_layer_07_O</zorder>
  <zorder>label_Environment_layer_06_O</zorder>
  <zorder>label_Environment_layer_06_I</zorder>
  <zorder>label_Environment_layer_05_I</zorder>
  <zorder>label_Environment_layer_05_O</zorder>
  <zorder>label_Environment_layer_04_O</zorder>
  <zorder>label_Environment_layer_04_I</zorder>
  <zorder>label_Environment_layer_03_O</zorder>
  <zorder>label_Environment_layer_03_I</zorder>
  <zorder>label_Environment_layer_02_I</zorder>
  <zorder>label_Environment_layer_02_O</zorder>
  <zorder>label_Environment_layer_01_O</zorder>
  <zorder>label_Environment_layer_01_I</zorder>
  <zorder>label_Enemy_Oculothorax</zorder>
  <zorder>label_Enemy_Slime</zorder>
  <zorder>label_Hero</zorder>
  <zorder>label_Score</zorder>
  <zorder>label_Enemy_Ogre_Boss</zorder>
  <zorder>label_Enemy_Ogre_Shockwave</zorder>
  <zorder>label_Environment_layer_00_O</zorder>
  <zorder>label_Environment_layer_00_I</zorder>
  <zorder>pushButton_label_menu</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>pushButton_open</zorder>
  <zorder>pushButton_score</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>pushButton_options</zorder>
  <zorder>pushButton_comboBox_ports</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
